packages/plugin-types-bundler/src/bundleTypes.ts

```diff
@@ -0,0 +1,50 @@
+import { getImportsInfo } from '@grafana/levitate';
+import { EntryPointConfig, generateDtsBundle } from 'jackw-dts-bundle-gen-test';
+import { writeFileSync } from 'node:fs';
+import { extname, join } from 'node:path';
+import { parsedArgs } from './utils.js';
+
+export const generateTypes = () => {
+  const { entryPoint, tsConfig, outDir } = parsedArgs;
+
+  if (!entryPoint) {
```

Comment:
**Nit:** although it has no harm, should we check it here as well? (We are already validating it in the `run.ts`.)